web/themes/new_weather_theme/templates/wfo-info/node--wfo-info.html.twig

```diff
@@ -1,41 +1,111 @@
+          {{ "Local expertise" | t }}
+        </h2>
+        <p>
+          Decision support is description tk tk tk
+        </p>
+        <ul class="padding-left-1">
+          <li class="padding-y-05"><a href="#">Briefing</a></li>
+          <li class="padding-y-05"><a href="#">County finder</a></li>
+          <li class="padding-y-05"><a href="#">State page</a></li>
+          <li class="padding-y-05"><a href="#">Resource 1</a></li>
+          <li class="padding-y-05"><a href="#">Resource 2</a></li>
+          <li class="padding-y-05"><a href="#">Resource 3</a></li>
+        </ul>
+      </div>
+
+      <div class="grid-col-12 tablet:grid-col-6">
+        <h2 class="border-top-1px border-base-light text-normal text-primary-darker padding-top-1">
+          {{ "Forecaster's discussion" | t }}
+        </h2>
+        <p>
+          The purpose of the area forecast discussion is to provide the
+          meteorological/scientific reasoning behind the forecast
+        </p>
+        <a href="#">Current Area Forecast Discussion</a>
+      </div>
+    </div>
+
+    {% set hasPhone = phone is not empty %}
+    {% set hasAddress = address is not empty %}
+    {% set hasEmail = email is not empty %}
+
+    {% set hasFacebook = not (content.field_facebook_url.isEmpty or content.field_facebook_url.0 is not defined) %}
+    {% set hasTwitter = not (content.field_twitter_url.isEmpty or content.field_twitter_url.0 is not defined) %}
+    {% set hasYoutube = not (content.field_youtube_url.isEmpty or content.field_youtube_url.0 is not defined) %}
+    {% set hasSocial = hasFacebook or hasTwitter or hasYoutube %}
+
+    {% set hasContact = hasSocial or hasAddress or hasPhone or hasEmail %}
+
+    {% if hasContact %}
+      <h2 class="border-top-1px border-base-light text-normal text-primary-darker padding-top-1 margin-bottom-0">
+        {{ "Contact us" | t }}
+      </h2>
+
+      <div class="grid-row margin-top-2">
+        <div class="grid-col-12 tablet:grid-col-4">
+          {% if hasPhone %}
+            {{ content.field_phone_number_opt }}
+          {% endif %}
+
+          {% if hasEmail %}
+            {% if hasPhone %}<div class="margin-top-2">{% endif %}
+              {{ content.field_wfo_email }}
+            {% if hasPhone %}</div>{% endif %}
+          {% endif %}
+
+          {% if hasAddress %}
+            {% if hasPhone or hasEmail %}<div class="margin-top-2">{% endif %}
+              {{ content.field_address }}
+            {% if hasPhone or hasEmail %}</div>{% endif %}
+          {% endif %}
+        </div>
+
+        <div class="grid-col-12 tablet:grid-col-8">
+          {% if hasSocial %}
+            <ul class="usa-list usa-list--unstyled"> 
+              {{ content.field_facebook_url }}
+              {{ content.field_twitter_url }}
+              {{ content.field_youtube_url }}
+            </ul>
+          {% endif %}
+        </div>
+      </div>
+    {% endif %}
+  </div>
+</div>
```

Comment:
As Eric notes there should be 10 units of space at the bottom of the page before the footer

Reply:
👍